test/mocks/mocks.go

```diff
@@ -433,8 +433,8 @@ func MultiNamespaceServiceBindingMock(
 			Name:      name,
 		},
 		Spec: v1alpha1.ServiceBindingSpec{
-			MountPathPrefix: "/var/redhat",
-			CustomEnvVar:    []corev1.EnvVar{},
+			MountPath:    "/var/redhat",
```

Comment:
Remove this line

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/735/commits/173e4233d503723751b0e24ec9f5bc6e428a4f29